### PR TITLE
Fixes to limiter for use in executables

### PR DESCRIPTION
**Pull request by fmahebert**
## Proposed changes

Small fixes needed to use limiters in an executable

### Types of changes:

- [x] Bugfix
- [ ] New feature

### Component:

- [x] Code
- [ ] Documentation
- [ ] Build system
- [ ] Continuous integration

### Code review checklist

- [ ] The PR passes all checks, including unit tests, `clang-tidy` and `IWYU`. For
  instructions on how to perform the CI checks locally refer to the [Dev guide
  on the Travis CI](https://spectre-code.org/travis_guide.html).
- [ ] The code is documented and the documentation renders correctly. Run `make doc`
  to generate the documentation locally into `BUILD_DIR/docs/html`. Then open
  `index.html`.
- [ ] The code follows the stylistic and code quality guidelines listed in the
  [code review guide](https://spectre-code.org/code_review_guide.html).

### CODE REVIEW

**Review by kidder — DISMISSED** [review]

File: src/Domain/CMakeLists.txt [hunk 1]
```diff
@@ -39,5 +39,6 @@ target_link_libraries(
   INTERFACE CoordinateMaps
   INTERFACE DataStructures
   INTERFACE ErrorHandling
+  INTERFACE LinearOperators
```
Comment (nilsdeppe) [inline src/Domain/CMakeLists.txt:42]: Just out of curiosity, what in domain needs linear operators?

File: src/Evolution/DiscontinuousGalerkin/SlopeLimiters/Minmod.hpp [hunk 2]
```diff
@@ -225,6 +225,11 @@ class Minmod<VolumeDim, tmpl::list<Tags...>> {
     tuples::TaggedTuple<Minmod_detail::to_tensor_double<Tags>...> means_;
     std::array<double, VolumeDim> element_size_ =
         make_array<VolumeDim>(std::numeric_limits<double>::signaling_NaN());
+
+    void pup(PUP::er& p) noexcept {
```
Comment (nilsdeppe) [inline src/Evolution/DiscontinuousGalerkin/SlopeLimiters/Minmod.hpp]: You'll need a `NOLINT` comment to silence clang-tidy

File: src/Domain/CMakeLists.txt (hunk 1 above)
Comment (fmahebert) [reply src/Domain/CMakeLists.txt:42]: The `size_of_element` function I added in PR #999 uses `mean_value_on_boundary`.

File: src/Evolution/DiscontinuousGalerkin/SlopeLimiters/Minmod.hpp (hunk 2 above)
Comment (fmahebert) [reply src/Evolution/DiscontinuousGalerkin/SlopeLimiters/Minmod.hpp]: Looks like clang-tidy passes on Travis... should I add the comment anyway?

Comment (nilsdeppe) [reply src/Evolution/DiscontinuousGalerkin/SlopeLimiters/Minmod.hpp]: Yes, that just means you're not using this code in a cpp file

**Comment by fmahebert** [comment]
@nilsdeppe I added the `NOLINT` and squashed

**Review by kidder — APPROVED** [review]

**Review by wthrowe — APPROVED** [review]

**Review by nilsdeppe — APPROVED** [review]

**Pull request merged**